source/core_http_client.c

```diff
@@ -1340,9 +1340,9 @@ static HTTPStatus_t addHeader( HTTPRequestHeaders_t * pRequestHeaders,
             pBufferCur += fieldLen;
 
             /* Copy the field separator, ": ", into the buffer. */
-            ( void ) strncpy( pBufferCur,
```

Comment:
Just leaving this here: This was the commit where we changed memcpy to strncpy to satisfy Coverity : 
https://github.com/FreeRTOS/coreHTTP/commit/e0ebc00a9f362160d9a7c6cbe159bdc213af3c67